.gitignore

```diff
@@ -5,3 +5,4 @@
 .idea
 node_modules
 npm-debug.log
+bower_components
```

Comment:
Yup, just including `bower_components` for now sounds right.  Longer term I'm wondering if the best approach would be to actually split this repo into two: one for the javascript library (and its associated template files, base css, etc) and a second for the hosted 350map.org site.  That seems like it would solve this problem (i.e. the 350map.org repo would include bower_components) and would probably let us clean up a bunch of other assorted ugliness in the codebase too like the bazillions of outdated index.html files that we can't really update without checking for backwards-compatibility (i.e. they'd just stay in the 350map.org repository and maybe even point at older builds of the js libraries) ... etc.  Would also almost certainly make it cleaner for other people to use the open source codebase.

Does that make sense to you, or do you think it would be overkill?

In any case, that's definitely a topic for another PR. :)